torch/onnx/symbolic_opset9.py

```diff
@@ -1044,11 +1052,18 @@ def __lshift_(g, self, other):
     return lshift
 
 
-def where(g, condition, self, other):
+def where(g, condition, self=None, other=None):
     # Assumes that torch.where's first argument takes only Bool and Byte tensors.
     if condition.type().scalarType() != 'Bool': 
         condition = g.op("Cast", condition, to_i=sym_help.cast_pytorch_to_onnx['Bool'])
-    return g.op("Where", condition, self, other)
+    if len(args) == 1:  # first arg is internal input "_output". self=None and other=None
+        _output = args[0]
+        condition = torch.onnx.symbolic_opset9.nonzero(g, condition)
+        return unbind(g, condition, g.op("Constant", value_t=torch.tensor(1)), _output)
+    else:  # first arg is self, and second arg is other
+        self = args[0]
+        other = args[1]
```

Comment:
is assigning values to self and other really necessary? are the values for self and other inferred when len(args)==2?
Can len(args) be 0?